tools/downloader/pytorch_to_onnx.py

```diff
@@ -58,10 +65,14 @@ def parse_args():
                         help='Space separated names of the output layers')
     parser.add_argument('--model-param', type=model_parameter, default=[], action='append',
                         help='Pair "name"="value" of model constructor parameter')
+    parser.add_argument('--onnx-version', type=int, default=9,
+                        help='Version of symbolic opset')
+    parser.add_argument('--prepare-params', type=bool, default=False,
```

Comment:
not clear argument name from my view, could you please give more descriptive name for it?